web3/_utils/module_testing/eth_module.py

```diff
@@ -141,6 +141,31 @@ async def test_eth_send_transaction_legacy(
         assert txn["value"] == 1
         assert txn["gas"] == 21000
         assert txn["gasPrice"] == txn_params["gasPrice"]
+    
+    @pytest.mark.asyncio
+    async def test_eth_sign_transaction(
+        self, w3: "Web3", unlocked_account: ChecksumAddress
```

Comment:
Looks like this should be: 
```suggestion
        self, async_w3: "Web3", unlocked_account: ChecksumAddress
```

Reply:
I guess the problem persisted